src/gobupload/relate/update.py

```diff
@@ -499,8 +502,7 @@ def _source_value_ref(self):
 
         if self.is_many:
             return f"{self.json_join_alias}.item->>'{FIELD.SOURCE_VALUE}'"
-        else:
-            return f"src.{self.src_field_name}->>'{FIELD.SOURCE_VALUE}'"
+        return f"src.{self.src_field_name}->>'{FIELD.SOURCE_VALUE}'"
```

Comment:
I'm not a big fan of removing the else-blocks everywhere. Having the indentation of the statements in the else-block the same as the indentation of the statements in the if-block makes it immediately clear that it's an if/else kind of situation. Removing the else-statement is often just harder to read. Same with if/elif/elif/elif/else or if/if/if/if/if. The use of elif, even when unnecessary, makes the intent of the code much clearer at first glance.